src/app/tests/main_test.py

```diff
@@ -0,0 +1,16 @@
+import unittest
```

Comment:
For the learning purpose please take a look on pytest: https://docs.pytest.org/en/stable/